packages/webpack-graph-ql/src/lib/types/warning.ts

```diff
@@ -0,0 +1,41 @@
+import { AutoSchema } from './auto-schema';
+
+type WarningData = {
+  message: string;
+  moduleId: string;
+  moduleIdentifier: string;
+  moduleName: string;
+  loc: string;
+  name: string;
+  warning: string;
+};
+
+// Resolvers for the "warning" field
+export class Warning extends AutoSchema {
+
+  @returns(String) message: string;
+  @returns(String) moduleId: string;
+  @returns(String) moduleIdentifier: string;
+  @returns(String) moduleName: string;
+  @returns(String) loc: string;
+  @returns(String) name: string;
+  @returns(String) warning: string;
+
+  @returns(Module)
+  get module(parent: any, args: any, context: any): Module {
```

Comment:
The mapping from parent, args, context needs to be put into the lift Resolver so the argument that the actual function takes is better representative of how just this fn would be called (ie we should have one argument "filePath". Probably passed in by the @query decorator